conda/exceptions.py

```diff
@@ -1297,16 +1297,24 @@ def __init__(self, msg: str, *args, **kwargs):
 
 
 class EnvironmentSpecPluginNotDetected(SpecNotFound):
-    def __init__(self, name: str, plugin_names: Iterable[str], *args, **kwargs):
+    def __init__(
+        self,
+        name: str,
+        plugin_names: Iterable[str],
+        autodetect_disabled_plugins: Iterable[str] = [],
+        *args,
+        **kwargs,
+    ):
         self.name = name
         msg = dals(
             f"""
-            Environment at {name} is not readable by any installed
-            environment specifier plugins.
+            Environment at {name} is not readable by any installed environment specifier plugins.
 
-            Available plugins: {dashlist(plugin_names, 4)}
+Available plugins: {dashlist(plugin_names, 4)}
```

Comment:
What's with the indentation here?

Reply:
oh, I see actually this can be fixed a better way by setting the right indentation on `dashlist`